scope

```diff
@@ -181,6 +187,7 @@ EOF
         if ! check_docker_for_mac || check_probe_only ; then
             check_not_running $SCOPE_CONTAINER_NAME $SCOPE_IMAGE_NAME
             docker rm -f $SCOPE_CONTAINER_NAME >/dev/null 2>&1 || true
+            create_plugins_dir
```

Comment:
Couldn't find an issue yesterday, but found this just now https://github.com/docker/docker/commit/249f45bcfe1adf4b3550306951f2768b17802217. Also, we are already handing this in Weave Net (https://github.com/weaveworks/weave/issues/1653).